Documentation/ITK5MigrationGuide.md

```diff
@@ -452,6 +452,9 @@ The namespece itself is only available in legacy mode. Some classes were renamed
 * `itk::SpatialOrientation::CoordinateMajornessTerms` became `itk::SpatialOrientationEnums::CoordinateMajornessTerms`.
 * `itk::SpatialOrientation::ValidCoordinateOrientationFlags` became `itk::SpatialOrientationEnums::ValidCoordinateOrientations`.
 
+Since ITK 5.3, `OrientationAdapterBase` is deprecated. It will be available in `ITKDeprecated` module.
```

Comment:
We should not be moving classes into different modules until a major release version.

Reply:
Force-push changes that.